tools/ARIAtools/set_beta_v1.py

```diff
@@ -300,6 +259,57 @@ def get_setLOS(self, inc_angle, az_angle):
 
         return tide_los
 
+    def update_gunw(self):
+        """ Update the path_gunw file using input correction layers """
+        # initialize cube from existing geometry field
+        with xr.open_dataset(self.path_gunw, group=self.geom_group) as ds_ifg:
+            ds_ifg[LYR_NAME] = ds_ifg.incidenceAngle
+            # drop geometry layers
+            ds_ifg = ds_ifg.drop(GEOM_LYRS)
+            # update variable + attributes
+            ds_ifg[LYR_NAME].values = self.tide_los
+            attrs = {
+                 'description': 'Solid Earth tide',
+                 'units': 'ray radians',
+                 'grid_mapping': 'crsMeta',
+                 'long_name': LYR_NAME,
+                 'standard_name': LYR_NAME
```

Comment:
When you branch this into its own repo it would be good to capture the versions of the softwares too.